src/tools/plotter/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(iDynTreePlotter)
+
+find_package(Qt5 COMPONENTS Widgets Charts)
+if (${Qt5_FOUND})
```

Comment:
Can you be more precise with the `OPTIONAL_DEPENDENCY` keyword? I am not able to find it in `iDynTreeDependencies.cmake`